client/src/components/App/Content/UsersManagement/useUsersManagement.ts

```diff
@@ -298,6 +303,34 @@ const useUsersManagement = ({ page, rowsPerPage, cellNameSort, setPage }: useUse
         }
     }
 
+    const handleDeactivateAllUsersCounty = () => {
+        alertWarning((deactivateAllCountyUsersWarningTitle),{
+            showCancelButton: true,
+            cancelButtonText: 'בטל',
+            cancelButtonColor: theme.palette.error.main,
+            confirmButtonColor: theme.palette.primary.main,
+            confirmButtonText: 'כן, המשך',
+        }).then((result) => {
+            if (result.value) {
+                const deactivateAllCountyUsersLogger = logger.setup('Updating all county users activity status to false');
+                deactivateAllCountyUsersLogger.info('send request to server for updating users activity statuses', Severity.LOW);
+                setIsLoading(true);
+                return axios.post('users/deactivateAllCountyUsers', {
+                    county: displayedCounty
+                }).then((result) => {
+                    if(result.data){
+                        fetchUsers();
+                        deactivateAllCountyUsersLogger.info('updated users activity statuses successfully', Severity.LOW); 
+                    }
+                }).catch((error) => {
+                    alertError('לא הצלחנו לכבות את כל החוקרים בנפה');
+                    deactivateAllCountyUsersLogger.error(`error in updating users activity statuses ${error}`, Severity.HIGH);
+                })
+                .finally(() => setIsLoading(false)); 
```

Comment:
Doesn’t need to be in finally, only in catch, because fetchUsers will turn the spinner off